File: CPP/Cap03/c03ex01.cpp
// c03ex01.cpp

#include <iostream>

using namespace std;

int main(void) {
  int A, B, X;

  cout << "Entre o 1o. valor numerico inteiro: ";
  cin >> A;
  cin.ignore();

  cout << "Entre o 2o. valor numerico inteiro: ";
  cin >> B;
  cin.ignore();

  X = A + B;

  cout << "Resultado = " << X << endl;

  cout << "\nTecle <Enter> para encerrar... ";
  cin.get();

  return 0;
}

File: DART/Cap03/c03ex01.dart
// c03ex01.dart

import 'dart:io';

void main() {
  int? A;
  int? B;
  int? X;

  stdout.write("Entre o 1o. valor numérico inteiro: ");
  A = int.tryParse(stdin.readLineSync() ?? '');

  stdout.write("Entre o 2o. valor numérico inteiro: ");
  B = int.tryParse(stdin.readLineSync() ?? '');

  X = A! + B!;
  print("Resultado = $X");

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
